backend/src/util/mapping/Mappings.js

```diff
@@ -243,7 +244,74 @@ const FacilityMappings = [
   { back: 'statecode', front: 'stateCode' },
   { back: 'statuscode', front: 'statusCode' },
   { back: '_ofm_primarycontact_value', front: 'primaryContactId' },
-  { back: 'ofm_program', front: 'programCode' },
+]
+const FacilityAdditionalAddressMappings = [
```

Comment:
I'm questioning whether it's worth the benefit of storing this as a Mapping if we're not using it to format for front. We could just generate the query string using a for loop, like how you handle the response.

Reply:
it's been moved out!